system-test/test.sh

```diff
@@ -55,17 +55,31 @@ if [[ "$VERIFY_TIME_LINE_NUMBERS" != "true" ]]; then
   npm run compile
 fi
 
+NODE_VERSION=$(node -v | cut -d. -f1 | tr -d 'v')
```

Comment:
```suggestion
NODE_VERSION=$(node -p "process.versions.node.split('.')[0]")
```

You can get the version without other cli tools

Reply:
Not necessary for this change set!